backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -34,6 +34,11 @@ body {
     @include margin(null auto);
     max-width: map-get($container-max-widths, "xl");
   }
+
+  &.max-width {
+    @include margin(null null auto null);
+    max-width: map-get($container-max-widths, "xl");
+  }
```

Comment:
This seems to duplicate `max-width-form`, or the other way around, but either way I think we should change the terminology, and either use the build-in `.container` or an adjusted derivative of it if we can't - like `.container.forms`, so to be clearer to others what the purpose of this class is, ie. it's already documented in Bootstrap.